File: camtrack_client/calcABCD.cpp
#include <iostream>  

using namespace std;

int main()
{
	double A,B,C,D;
	
	double dh1 = 6;
	double dv1 = 14;
	double len1 = 32.0156;

	double dh2 = 7;
	double dv2 = 31;
	double len2 = 67.897;

	A = (dh2-dh1)/(len2-len1);
	B = dh1-len1*A;
	C = (dv2-dv1)/(len2-len1);
	D = dv1-len1*C;

	cout<<"A: "<<A<<"    B: "<<B<<"    C: "<<C<<"    D: "<<D<<endl;
	
	cin>>A;
	return 1;

}

File: camtrack_server/oth tools/calcABCD.lua



local A,B,C,D;

local dh1  = 29;
local dv1  = -6;
local len1 = 33;

local dh2  = 33;
local dv2  = -1;
local len2 = 55;



function calcABCD()

	A = (dh2-dh1)/(len2-len1);
	B = dh1-len1*A;
	C = (dv2-dv1)/(len2-len1);
	D = dv1-len1*C;

	print("A: "..A.."    B: "..B.."    C: "..C.."    D: "..D);


end

return calcABCD();
